src/content/docs/en/guides/deploy/aws.mdx

```diff
@@ -113,77 +113,22 @@ CloudFront is a web service that provides content delivery network (CDN) capabil
-        "Action": "s3:GetObject",
-        "Resource": "arn:aws:s3:::astro-aws/*"
-      }]
-    }
-    ```
-
-    :::caution
-    Do not forget to replace `<CLOUDFRONT_OAI_ID>` with the name of your CloudFront Origin Access Identity ID. You can find the CloudFront Origin Access Identity ID in **CloudFront > Origin access identities** after setting up CloudFront.
-    :::
-
 ### CloudFront setup
 
 1. Create a CloudFront distribution with the following values:
     * **Origin domain:** Your S3 bucket
-    * **S3 bucket access:** "Yes use OAI (bucket can restrict access to only CloudFront)"
+    * **Origin access:** "Use *Legacy access identities* (bucket can restrict access to only CloudFront)"
     * **Origin access identity:** Create a new origin access identity
-    * **Viewer - Bucket policy:** "No, I will update the bucket policy"
+    * **Bucket policy:** "Yes, update the bucket policy"
     * **Viewer protocol policy:** "Redirect to HTTPS"
-    * **Default root object:** `index.html`
-
-This configuration will block access to your S3 bucket from the public internet and serve your site using the global CDN network. You can find your CloudFront distribution URL in the bucket's **Distributions > Domain name**.
-
-### CloudFront Functions setup
 
-Unfortunately, CloudFront does not support multi-page `sub-folder/index` routing by default. To configure it, we will use CloudFront Functions to point the request to the desired object in S3.
-
-1. Create a new CloudFront function with the following code snippet. You can find CloudFront functions in **CloudFront > Functions**.
-
-    ```js
-    function handler(event) {
-      var request = event.request;
-      var uri = request.uri;
-
-      // Check whether the URI is missing a file name.
-      if (uri.endsWith('/')) {
-        request.uri += 'index.html';
-      } 
-      // Check whether the URI is missing a file extension.
-      else if (!uri.includes('.')) {
-        request.uri += '/index.html';
-      }
+  :::caution
+  After selecting your S3 bucket as origin domain, CloudFront will recommend using the S3 website endpoint instead.
+  Click `Use website endpoint`, otherwise requests may not be resolved correctly.
+  If the banner does not show, your S3 bucket is missing the website hosting endpoint, which can be configured for the bucket under **Settings > Static website hosting**.
+  :::
 
-      return request;
-    }
-    ```
-  2. Attach your function to the CloudFront distribution. You can find this option in your CloudFront distribution's **Settings > Behaviors > Edit > Function** associations.
-        * **Viewer request - Function type:** CloudFront Function.
-        * **Viewer request - Function ARN:** Select the function you created in the previous step.
+This configuration will block access to your S3 bucket from the public internet and serve your site using the global CDN network. You can find your CloudFront distribution URL in the bucket's **Distributions > Domain name**.
```

Comment:
Unfortunately S3 policies control access to cloudfront too so there is no way to prevent access to S3 bucket.
```suggestion
This configuration will serve your site using the global CDN network. You can find your CloudFront distribution URL in the bucket's **Distributions > Domain name**.
```